src/igd_async.rs

```diff
@@ -130,7 +130,28 @@ quick_error! {
     }
 }
 
-pub fn get_any_address(
+pub fn get_any_address_rendezvous(
+    protocol: Protocol,
+    local_addr: SocketAddr,
+    timeout: Duration,
+    handle: &Handle,
+) -> BoxFuture<SocketAddr, GetAnyAddressError> {
+    if !is_igd_enabled_for_rendezvous() {
+        return future::err(GetAnyAddressError::Disabled).into_boxed();
+    }
+
+    get_any_address(protocol, local_addr, Some(timeout), handle)
+}
+
+pub fn get_any_address_open(
```

Comment:
It took me some time to understand the difference between `get_any_address_open()` and `get_any_address_rendezvous()`.
It would be nice to have some comments on each or more descriptive naming, e.g. `get_any_address_for_server`, etc.